pkg/models/active_node.go

```diff
@@ -2,10 +2,11 @@ package models
 
 //ActiveNode model
 type ActiveNode struct {
-	Id        int64  `json:"id"`
-	NodeId    string `json:"node_id"`
-	Heartbeat int64  `json:"heartbeat"`
-	Sequence  int32  `json:"sequence"`
+	Id           int64  `json:"id"`
+	NodeId       string `json:"node_id"`
+	Heartbeat    int64  `json:"heartbeat"`
+	Sequence     int32  `json:"sequence"`
```

Comment:
We can change the name to partition_no as this is not an id but a partition no for that time interval